lists/LaetisShoppingList.Md

```diff
@@ -1,9 +1,3 @@
-milk
-raisin bread
-fresh spinach leaves
-cheese
 dog food
-wet wipes
-potatoes
 graded cheese
```

Comment:
* Missing a "-"
* Typo: "grated"

Reply:
@pbuttigieg please see my [edits](1bdb9535e20e58ae3063af85fadc98ee8f4501cf). I grated the cheese and added the lines.